scripts/update-ctags.py

```diff
@@ -13,18 +13,21 @@
 dstdir = os.path.abspath(sys.argv[2])
 
 os.chdir(dstdir + '/parsers')
-parser_dst_files = glob.glob('**/*.c') + glob.glob('**/*.h')
+parser_dst_files = glob.glob('*.c') + glob.glob('*.h')
 parser_dst_files = list(filter(lambda x: not x.startswith('geany_'), parser_dst_files))
-os.chdir(srcdir)
+os.chdir(srcdir + '/parsers')
 print('Copying parsers... ({} files)'.format(len(parser_dst_files)))
 for f in parser_dst_files:
-    if os.path.exists('parsers/' + f):
-        shutil.copy('parsers/' + f, dstdir + '/parsers/' + f)
-    elif os.path.exists(f):
-        shutil.copy(f, dstdir + '/parsers/' + f)
-    else:
-        print('Error: Could not find file ' + f + '!')
-        sys.exit(1)
+    shutil.copy(f, dstdir + '/parsers')
+
+os.chdir(dstdir + '/peg')
+peg_dst_files = glob.glob('*.c') + glob.glob('*.h')
+peg_dst_files = list(filter(lambda x: not x.startswith('geany_'), peg_dst_files))
+os.chdir(srcdir + '/peg')
+os.system('make -C {} peg/{}'.format(srcdir, ' peg/'.join(peg_dst_files)))
+print('Copying peg parsers... ({} files)'.format(len(peg_dst_files)))
+for f in peg_dst_files:
+    shutil.copy(f, dstdir + '/peg')
```

Comment:
Since the code does more or less the same thing twice, it could be changed to something like the following:

```
os.chdir(srcdir + '/peg')
os.system('make -C {} peg/{}'.format(srcdir, ' peg/'.join(peg_dst_files)))
for dir in ['parsers', 'peg']:
    ...
```

Reply:
I was actually thinking that the script could use some "readability improvements". Like using functions and giving it some structure... But it seemed a bit out of scope for this PR, so I stayed with the current style. 

Would you mind if I implemented it in follow-up PR?

Reply:
> Would you mind if I implemented it in follow-up PR?

Yep, agreed. #3032 will do some more changes to the script so maybe better wait until both of these pull requests are merged.